packages/react-codemirror/src/e2e_tests/signatureHelp.spec.tsx

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable @typescript-eslint/unbound-method */
```

Comment:
When building from VSCode, the linter was complaining about the ` async ({ page, mount }) => {` parts in the tests